lnd.go

```diff
@@ -1195,8 +1142,7 @@ func waitForWalletPassword(cfg *Config, restEndpoints []net.Addr,
 	for _, lis := range listeners {
 		wg.Add(1)
 		go func(lis *ListenerWithSignal) {
-			rpcsLog.Infof("Password RPC server listening on %s",
-				lis.Addr())
+			rpcsLog.Infof("RPC server listening on %s", lis.Addr())
 
 			// Close the ready chan to indicate we are listening.
 			close(lis.Ready)
```

Comment:
Because we now start both listeners at the same time, the `Ready` is closed immediately. We rely on this to only be closed on the main RPC listener. Perhaps we now don't need two separate listeners anymore and can simplify this as well? Would require more refactoring and updates to the falafel template. But it looks like you already started some of this in #5010. We just need two signals (ready to be unlocked and ready for main operation) on the same listener.

Reply:
Yeah, the second listener is removed in #5010. 

It is hard to remove it in this PR since it will break the mobile build, and that change relies on a `falafel` change. But if you want to test this , should be able to test #5010 directly, as it build on top of this!

Reply:
Ah, I completely missed the new functionality in #5005. Very cool!
I added a PR to `lndclient` and all other projects and was able to verify that this works well in `lightning-terminal`: https://github.com/lightninglabs/lndclient/pull/41
Awesome, this makes it so much easier and cleaner too :tada: